package.json

```diff
@@ -47,6 +47,6 @@
     "node": ">=18.0.0 <19.0.0"
   },
   "dependencies": {
-    "@wordpress/icons": "^9.36.0"
+    "@wordpress/icons": "^9.31.0"
```

Comment:
what was the reason we went down in version

Reply:
Unknown. Build for production runs npm ci, but that shouldn't write to the package.json... I did not explicitly modify this.